newsApp.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const app = express();
+
+const {
+    handle500Statuses,
+    handlePSQL400s,
+    handleCustomErrors,
+} = require('./newsControllers/errorHandlingControllers');
+
+app.use(express.json());
```

Comment:
We do not need to parse any json in the app yet so this middleware should not yet be included